woocommerce/woocommerce-android
# Issue/5086 add icons for ipp options

Closes: #5086 


### Description

This PR adds icons for the In-Person Payments menu. The reasoning behind this can be found here p1635231687060500-slack-C025A8VV728

in-short, the existing menu lacks the distinction between different options in the menu

### Note to reviewer
I will be raising another PR targeting `epic/stripe-sdk-2.0` which contains `Card Reader Manual icon` for `Stripe M2 Card Reader` option

### Testing instructions

1. Navigate to the `settings` screen
2. Open the `In-Person Payments` screen
3. Make sure the icons are visible for every option

### Images/gif
<p float="left">
<img src=https://user-images.githubusercontent.com/1331230/139251380-8e0c2e4b-7cfc-4e78-b05b-9c68fd58bf89.png width=250px/>


<img src=https://user-images.githubusercontent.com/1331230/139251479-95fe33a3-a4ee-4528-bbc8-5cd213ec6acd.png width=250px/>
</p>
